src/components/MainSettingsScreen/HeaderAddButton.tsx

```diff
@@ -15,9 +15,10 @@ export class HeaderAddButton extends React.Component<HeaderAddButtonProps> {
     public render(): ReactNode {
         return (
             <Button
-                style={styles.topRightButton}
+                type="clear"
+                buttonStyle={styles.topRightButton}
                 onPress={this.props.onPress.bind(this)}
-                icon={<Icon name="add" type="ionicons" />}
+                icon={<Icon name="add" type="ionicons" color="#007AFF" />}
```

Comment:
Could we get this added to `constants/Colors.ts`?

Reply:
Will do!